content/300-guides/300-upgrade-guides/800-upgrade-from-prisma-1/01-how-to-upgrade.mdx

```diff
@@ -17,7 +17,7 @@ The upgrade documentation consists of several pages, here's an overview of how t
 
 In addition to these two pages, there are various _practical guides_ that walk you through an example scenario of the upgrade process:
 
-- [Upgrading the Prisma layer](/guides/upgrade-guides/upgrade-from-prisma-1/upgrading-the-prisma-layer-postgres): No matter what your Prisma 1 setup looks like, you should **always start your upgrade process by following this guide**.
+- [Upgrading the Prisma layer](/guides/upgrade-guides/upgrade-from-prisma-1/upgrading-the-prisma-layer-postgres-postgres-postgres-postgres-postgres): No matter what your Prisma 1 setup looks like, you should **always start your upgrade process by following this guide**.
```

Comment:
This look broken 😆

Reply:
Ha! that was a logic glitch 😆 I have fixed those now